resources/sass/_base.scss

```diff
@@ -165,77 +163,77 @@ $caret-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5
   // Other styles
   var,
   var a {
-    color: $variable-type-color;
+    color: var(--variable-type-color);
     font-style: normal;
   }
 
   dt:hover var,
   dt:hover var a {
-    color: $variable-type-color-hover;
+    color: var(--variable-type-color-hover);
   }
 
   dfn {
     font-style: normal;
     font-family: monospace;
-    color: $variable-name-color;
+    color: var(--variable-name-color);
   }
 
   pre {
-    color: $text-color;
-    margin: 0 0 0 ($spacing * 3px);
+    color: var(--text-color);
+    margin: 0 0 0 calc(var(--spacing) * 3);
     padding: 5px;
     overflow-y: hidden;
     border-top: 0;
-    border: $border;
-    background: $main-background;
+    border: var(--border);
+    background: var(--main-background);
     display: block;
     word-break: normal;
   }
 
   .kint-access-path-trigger,
   .kint-folder-trigger,
   .kint-search-trigger {
-    background: rgba($text-color, 0.8);
+    background: color-mix(in srgb, var(--text-color) 80%, transparent);
     border-radius: 3px;
-    height: $color_size * 1px;
-    font-size: $color_size * 1px;
+    height: var(--color-size);
+    font-size: var(--color-size);
     margin-left: 5px;
     font-weight: bold;
-    width: $color_size * 1px;
+    width: var(--color-size);
     text-align: center;
     float: right !important;
     cursor: pointer;
-    color: $main-background;
+    color: var(--main-background);
     position: relative;
     overflow: hidden;
 
     // Carefully balanced on multiple browsers
-    line-height: $color_size * 1.1 * 1px;
+    line-height: calc(--color-size * 1.1);
 
     &:hover {
-      color: $text-color;
-      background: $main-background;
+      color: var(--text-color);
+      background: var(--main-background);
     }
   }
 
   // The &telrec; icon is slightly smaller than the others,
   // so make it a bit bigger to line it out better
   .kint-search-trigger {
-    font-size: $color_size * 1.25 * 1px;
+    font-size: calc(--color-size * 1.25);
```

Comment:
Broken. Missing `var()`

All of this stuff was carefully tweaked numbers while checking browsers but it looks like `calc` behaves differently and it'll need to be straightened out afterwards

Reply:
I must apologize again, I really don't know why I missed some of these 😅 

Anyways, piece of advice (~15 years of web dev) is that using this approach is not a sure thing across all the different screens, DPI scalings, fonts, OSes, etc.

Can you please include of how it is supposed to look? I believe we can include a SVG icon that would look way better, anyways.

Reply:
The problem is that we have 3 icons on the right:

* ⌕
* ↧
* ⇄

I don't know about your screen but on mine one of these is much smaller than the rest:

> ![image](https://github.com/user-attachments/assets/78f9bf90-8b88-4d76-bd8c-af7279a78a1f)

It could just be my font but 1.25 has been pretty consistent for me

Reply:
I am not applying any custom fonts or anything like that, but the magnifying glass on my screen clearly has the same width as other icons, while in your screenshot it does not:

![image](https://github.com/user-attachments/assets/11773286-986d-4fee-8510-ac3bc593f67f)

I vote to use SVG icons here too. If you accept, I will whip a quick PR that uses Sass to load SVG icons directly into the CSS code.

One advantage this has is better theming support, like how the caret is themed.

Not lying, I disliked how these icons looked right from start when they were introduced, they don't "feel" like the rest of Kint, and I believe they can be improved if SVG icons are used instead.

Reply:
Alrighty, whip me up an SVG then. I wrote the one in the caret by hand years ago.

I know you can make an svg spritesheet where each icon is on its own "layer" but I don't remember how to do it.

As far as the icons go other than the search icon (Which is pretty universally a magnifying glass) I have no particular preference. The ones in use were just easily usable as htmlentities.